packages/ibm-products/src/components/Datagrid/useSelectRows.tsx

```diff
@@ -119,6 +119,10 @@ const SelectRow = (datagridState) => {
       getRowId,
       selectAll: null,
     });
+    // focus the radio / checkbox if lost
+    const activeElement = document.activeElement?.id ?? '';
```

Comment:
Gotta be careful to watch out for SSR apps where document will be `undefined`. You could add an optional chain here or save the active element in a ref from `useIsomorphicEffect`.